src/ampJson.js

```diff
@@ -1,7 +1,14 @@
-export default function ampJson(data, component) {
+export default function ampJson(data, component, attributes={}) {
   if (!data || !component) return '';
+
+  const attributesString = attributes ?
+    Object.entries(attributes).map(([key, val]) => (
+      `${key}=${val}`
+    )).join(' ').trim() :
+    undefined;
+
   return Object.entries(data).map(([key, value]) => (
-    `<${component} id="${key}">
+    `<${component} id="${key}" ${attributesString}>
```

Comment:
@ljharb Ok. How about instead of using `ampJson` for different components, I break it down into separate components:

```
ampState.json
ampAnalytics.json
```

and then ampAnalytics will take an optional param for `type` ?